src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -37,7 +38,7 @@ class FOSHttpCacheExtension extends Extension
     /**
      * {@inheritdoc}
      */
-    public function getConfiguration(array $config, ContainerBuilder $container)
+    public function getConfiguration(array $config, ContainerBuilder $container): ?ConfigurationInterface
```

Comment:
```suggestion
    public function getConfiguration(array $config, ContainerBuilder $container): ConfigurationInterface
```

i think we are allowed to be more strict about the return type than the parent class. we know we always return an instance and never null